R/epi_df.R

```diff
@@ -309,3 +323,7 @@ as_epi_df.tbl_ts <- function(x, as_of, other_keys = character(), ...) {
 is_epi_df <- function(x) {
   inherits(x, "epi_df")
 }
+
+group_epi_df <- function(x) {
+  x %>% group_by(group_by(across(all_of(kill_time_value(key_colnames(.))))))
```

Comment:
```suggestion
  x %>% group_by(across(all_of(kill_time_value(key_colnames(.)))))
```
suggest: more descriptive name; if still trying to be short, maybe `epigroup_epi_df`